.github/workflows/tox.yml

```diff
@@ -0,0 +1,32 @@
+---
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install tox tox-gh-actions
```

Comment:
Probably should be --user, but who cares :)

Reply:
Exactly.